docs/conf.py

```diff
@@ -66,8 +66,38 @@
     # TODO: fix is to install fonts in CI or remove opengraph
     # "sphinxext.opengraph",
     "sphinx_inline_tabs",
+    # Inheritance diagrams
+    "sphinx.ext.graphviz",
+    "sphinx.ext.inheritance_diagram",
 ]
 
+# Inheritance diagram settings - you can find various settings in the
+# graphviz docs here https://graphviz.org/doc/info/attrs.html
+inheritance_graph_attrs = dict(
```

Comment:
This is where you set style settings and such for the diagrams. 

NOTE - we can also generate diagrams [manually using graphviz](https://chiplicity.readthedocs.io/en/latest/Using_Sphinx/UsingGraphicsAndDiagramsInSphinx.html) if we wish!